tad_core/main.py

```diff
@@ -36,10 +36,15 @@ def main():
             shap_tool = ShapTool(model, data)
             shap_values = shap_tool.get_results()
             shap_tool.save_results(shap_values, args.outputdir)
-        case ArgParser.Actions.QUESTIONNAIRE:
-            from tad_core.tools.questionnaire_tool import QuestionnaireTool
```

Comment:
it is general best practive to have all imports at the top

Reply:
I understand. It turns out though, that our tool is starting to use a lot of imports. When all imports, especially those needed to perform a TAD test are loaded, the tool becomes slow at startup. This was mentioned by @anneschuth, which is why I moved imports to the place 'when they are needed'. I am aware this is not best practice, then again it is nice to only load what is needed when it is needed..